src/veins/modules/messages/WaveShortMessage.msg

```diff
@@ -42,4 +42,8 @@ packet WaveShortMessage {
 	int recipientAddress = -1;
 	int serial = 0;
 	simtime_t timestamp = 0;
+
```

Comment:
Is a new field to indicate a unicast message really necessary? With the existing recipientAddress, one could use a special value such as -1 to denote broadcasts with all other values being unicast addresses.